src/args.rs

```diff
@@ -41,6 +41,10 @@ pub struct Args {
     /// User agent to be used to send requests
     #[structopt(short, long, default_value = "suckit", help="User agent to be used for sending requests")]
     pub user_agent: String,
+
+    /// Wildcards to filter download of pages to only matched ones
+    #[structopt(short, long, default_value = ".*", help="Wildcards to filter download of pages to only matched ones")]
+    pub wildcards: String,
```

Comment:
Not a big fan of the wildcards name. This is a regex after all, so why not ```disk_save_accept_regex``` (or similar)

Reply:
With the additional exclude flag. these flags will be changed to `--exclude` and `--include`. Are we okay with that?

Reply:
Ok for me